<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>200</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>200</number>
    </property>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="welcomeText">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>32</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string> Welcome To Blackjack</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="levelOne">
        <property name="font">
         <font>
          <pointsize>43</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Level 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="levelTwo">
        <property name="font">
         <font>
          <pointsize>43</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Level 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="levelThree">
        <property name="font">
         <font>
          <pointsize>43</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Level 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="levelFour">
        <property name="font">
         <font>
          <pointsize>43</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Level 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
